ext/mbstring/tests/bug43994.phpt

```diff
@@ -49,7 +49,8 @@ With $regs arg:
 
 Warning: mb_ereg(): empty pattern in %s on line %d
 bool(false)
-NULL
+array(0) {
+}
```

Comment:
> Nonetheless, the type declaration of the $regs parameter in the function prototype is array, what doesn't match the current behavior. However, the function prototype doesn't match the function's behavior anyway, what can be observed with strict_types=1.

@cmb69 Sorry I don't understand exactly what you mean. Should I close this PR or do you think it is useful ?